AB_environments/AB_baseline.conda.yaml

```diff
@@ -30,23 +29,19 @@ dependencies:
   - xarray ==2023.8.0
   - zarr ==2.16.1
   - cftime ==1.6.2
-  - msgpack-python ==1.0.5
+  - msgpack-python ==1.0.6
   - cloudpickle ==2.2.1
   - tornado ==6.3.3
   - toolz ==0.12.0
   - zict ==3.0.0
   - xgboost ==1.7.6
   - optuna ==3.3.0
   - scipy ==1.11.2
-  - snowflake-connector-python ==3.2.0
-  - snowflake-sqlalchemy ==1.5.0
-  - sqlalchemy ==1.4.49  # Pinned by snowflake-sqlalchemy
+  - sqlalchemy ==2.0.21
   - pynvml ==11.5.0
   - bokeh ==3.2.2
   - gilknocker ==0.4.1
   - openssl >1.1.0g
-  - pyopenssl ==23.2.0
-  - cryptography ==41.0.4
```

Comment:
These two were pinned because in the past caused a fuss with snowflake. This doesn't seem to be the case anymore.